### PR TITLE
feat(TopBar): show description during call

### LINKED ISSUE #10562: Conversation description not shown during call
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
Currently the description only shows when you are in the chat:
image

But not when you are in the call:
image

**Describe the solution you'd like**
It would be nice if the description also shows during a call, using the subline display as it does in the chat.

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #10562

### 🖼️ Screenshots / Screencasts

| 🏚️ Before | 🏡 After |
|------------|----------|
|image      | image    |

### 🚧 Tasks

- [x] Design review

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Comment by Antreesy** [comment]
Was introduced here: https://github.com/nextcloud/spreed/pull/8341/commits/e7f3388db737d0f93e0797a9e1961a85622de203
So I'd ask @marcoambrosini , why it has been done back then?

**Comment by marcoambrosini** [comment]
> So I'd ask @marcoambrosini , why it has been done back then?

 It was done to minimize information that was not crucial during a call. Actually @jancborchardt back then you were in favor of reducing text in the call view. 
Also, as far as I can remember, no call view from other apps shows conversation or meeting description while in the call.

**Review by nickvergessen — APPROVED** [review]
We didn't add it because we wanted to put information like "people in call", "call duration", etc. there, but they are now in another place, so we can show the description there now.

**Comment by marcoambrosini** [comment]
> We didn't add it because we wanted to put information like "people in call", "call duration", etc. 

That proposal came later on, avatar and name were there before call time was added.

**Review by jancborchardt — CHANGES_REQUESTED** [review]
@DorraJaouad nice! Just one thing: It looks like the text doesn’t have enough contrast, since it’s color-text-maxcontrast which is intended to be used on a light background but it’s used on dark. Either:
- Is is possible to invoke the dark-theme color-text-maxcontrast for the subline here?
- Or we use color-main-text for the subline here too, when in a call

@marcoambrosini Whatever was discussed in the past, most recent consensus was to add the description in the subline, same as how it is in the chat and as per the issue: https://github.com/nextcloud/spreed/issues/10562

**Comment by nickvergessen** [comment]
> Is is possible to invoke the dark-theme color-text-maxcontrast for the subline here?

https://github.com/nextcloud/spreed/blob/c74c6f199a18aed32002a74bf36c47b98a37f49f/src/components/TopBar/TopBar.vue#L501

**Comment by jancborchardt** [comment]
> > Is is possible to invoke the dark-theme color-text-maxcontrast for the subline here?
> 
> https://github.com/nextcloud/spreed/blob/c74c6f199a18aed32002a74bf36c47b98a37f49f/src/components/TopBar/TopBar.vue#L501

This is a deprecated variable though. Only text colors we should use are `color-main-text` and `color-text-maxcontrast` as per design guidelines: https://docs.nextcloud.com/server/latest/developer_manual/design/foundations.html#text-color

I'd say for simplicity and to make sure the contrast works, let's use `color-main-text` for the subline here too.

--- 

@Antreesy yes, the contrast works with dark theme enabled because the `color-text-maxcontrast` changes shade slightly. This is why I suggested: 

> Is it possible to invoke the dark-theme color-text-maxcontrast for the subline here?

**Comment by Antreesy** [comment]
> This is a deprecated variable though

Then we should also migrate to `color-text-maxcontrast` in some places.
For the `color-main-text` for a description, I can't see, why not. Conversation name is bold, so would be distinguishable enough

**Comment by DorraJaouad** [comment]
> let's use color-main-text for the subline here too.


It conveyed 'too much' contrast to me so I opted for the dark theme `color-text-maxcontrast`
image

But I set it to `color-main-text` as suggested

File: src/components/TopBar/TopBar.vue [hunk 1]
```diff
@@ -39,17 +39,19 @@
 					{{ conversation.displayName }}
 				</p>
 				<p v-if="showUserStatusAsDescription"
-					class="description">
+					class="description"
+					:class="{'description__in-chat' : !isInCall }">
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/TopBar/TopBar.vue:43]: chat : 
image

call:
image

**Review by jancborchardt — APPROVED** [review]
Looks good according to screenshots, and thanks for the deprecated variable fixes! :)

**Comment by marcoambrosini** [comment]
Can we at least go back to fading things out after some mouse inactivity?

**Pull request merged**